src/Core/Models/Response/ConfigResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Bit.Core.Models.Response
+{
+    public class ConfigResponse
+    {
+        public static string DisplayEuEnvironmentFlag = "display-eu-environment";
```

Comment:
@mp-bw any better place to store feature flags? Thinking `Constants.cs` or it's own static class.

Reply:
Agreed `Constants.cs` would be the proper place for it to follow existing patterns

Reply:
https://github.com/bitwarden/mobile/pull/2454#discussion_r1152100129
@fedemkr suggested it was moved to the ConfigResponse.

Reply:
@andrebispo5 @fedemkr Since it's `public` it seems like `Constants` would be the right place for it.  Can it be `private`?

Reply:
Nope it needs to be public as it is going to be used as the key to retrieve the flag from the dictionary. I'll move it to the Constants file.